cmd/scheduler.go

```diff
@@ -120,6 +120,9 @@ func addCommands() {
 
 	// Backups
 	add(true, "Backup", "backup", daily, 12*hour, "ruby", "c:\\imqsbin\\cronjobs\\backup_v8.rb").SetStartTime(23, 0)
+
+	//Pinging
```

Comment:
I know this seems petty, but we do leave a space after //

```go
//Bad
// Good
```

If we're not strict about the little things, we end up with "broken windows" syndrome: See https://en.wikipedia.org/wiki/Broken_windows_theory